tests/perf/fair_queue_perf.cc

```diff
@@ -32,19 +32,19 @@
 static constexpr fair_queue::class_id cid = 0;
 
 struct local_fq_and_class {
-    seastar::fair_group fg;
+    seastar::shared_throttle fg;
```

Comment:
Minor: In the next patch we change the name of `_fgs` to `_throttle` in io_queue. I am wondering -- do we want to adjust the names also in fair_queue's tests?